chore: update to 2.23.0

xoascf opened the pull request:
Requires https://github.com/deepin-community/yyjson/pull/4.

xoascf committed:
chore: update to 2.23.0

UTsweetyfish commented on debian/changelog:
```diff
@@ -1,3 +1,27 @@
+fastfetch (2.23.0-0deepin) unstable; urgency=medium
```
```suggestion
fastfetch (2.23.0-0deepin1) unstable; urgency=medium
```

UTsweetyfish reviewed (changes requested):
Please bump VER2 to 1.

xoascf committed:
Fix lintian

latest-changelog-entry-without-new-date
debian-changelog-line-too-long
synopsis-too-long

xoascf committed:
Update changelog

xoascf committed:
Update control

Use libyyjson-dev instead of yyjson-dev

UTsweetyfish commented:
/topic fastfetch-2.23.0

UTsweetyfish commented:
/integr-topic fastfetch-2.23.0

UTsweetyfish commented:
/approve
